javascript2.jade/src/org/netbeans/modules/javascript2/jade/resources/JadeExample.jade

```diff
@@ -1,3 +1,23 @@
+<!--
```

Comment:
The name hints, that the file is used as a display sample - if it is for example used to demonstrate syntax highlighting, we added a comment type `GUI_USABILITY` (Code is visible at runtime and added license would negatively affect usability). This might qualify.

Reply:
Valid point; will provide `licenseinfo.xml` for the file.